charts/agent/README.md

```diff
@@ -342,8 +347,12 @@ Chart to install K8s collection stack based on Observe Agent
 | node.containers.logs.retryOnFailure.maxInterval | string | `"30s"` |  |
 | node.containers.logs.startAt | string | `"end"` |  |
 | node.containers.metrics.enabled | bool | `true` |  |
+| node.containers.metrics.interval | string | `"60s"` |  |
 | node.enabled | bool | `true` |  |
 | node.metrics.enabled | bool | `true` |  |
+| node.metrics.fileSystem.excludeMountPoints | string | `"[\"/dev/*\",\"/proc/*\",\"/sys/*\",\"/run/k3s/containerd/*\",\"/var/lib/docker/*\",\"/var/lib/kubelet/*\",\"/snap/*\"]"` |  |
+| node.metrics.fileSystem.rootPath | string | `"/hostfs"` |  |
```

Comment:
what does this depend on?

Reply:
This was a list I picked up from the otel presets results.  It's been in the background the whole time - was bringing it forward as it seems like a control that will be good to make configurable.